tools/biomodelsML/bio_ml_tool.xml

```diff
@@ -0,0 +1,89 @@
+<tool id="bio_ml_tool" name="Random Forest model" version="0.0.1" profile="22.05">
```

Comment:
```suggestion
<tool id="biomodel_BIOMD0000001066" name="Random Forest model" version="1" profile="22.05">
```

I have taken those from https://www.ebi.ac.uk/biomodels/BIOMD0000001066#History

Reply:
sounds good, fixed it.